packages/core/src/util.ts

```diff
@@ -38,4 +38,22 @@ export const validateImageExtension = (url: string) => {
 		throw new DiscordComponentsError(`The url of an image for discord-image-attachment should match the regex ${IMAGE_EXTENSION}`);
 };
 
-export const getGlobalEmojiUrl = (emojiName: string): Emoji | undefined => getConfig().emojis?.[emojiName];
+export const getGlobalEmojiUrl = (emojiName: string): Emoji | undefined => {
+	const globalEmoji = getConfig().emojis?.[emojiName];
+	if (globalEmoji) return globalEmoji;
+
+	const emojiRegex = /<a?:(?<name>\w+):(?<id>\d+)>/;
```

Comment:
```suggestion
	const emojiRegex = /(?:<(?<animated>a)?:(?<name>\w{2,32}):)?(?<id>\d{17,21})>?/;
```

As taken from https://github.com/sapphiredev/utilities/blob/d8b3b84b96f10031466955e5c272337fa95e6933/packages/discord-utilities/src/lib/regexes.ts#L42

Reply:
Agreed, forgot it's an option.